docs/modules/automation-testing/pages/scripting/auto-generate-an-appium-script.adoc

```diff
@@ -10,9 +10,9 @@ Learn how to generate an Appium script from a manual session, so you can start b
 [#_export_your_script]
 == Export your script
 
-After creating the baseline session, search for your session, open the session overview, and select *Automated test case*.
+After creating the baseline session, search for your session, open the session overview, and select *Convert to Test Case*.
```

Comment:
Should it point to [this guidance](https://kobiton.atlassian.net/wiki/spaces/KOBITON/pages/3630071929/Convert+session+to+test+case+Review#Create-a-test-case-in-the-Test-Management) instead of from a session?